nifi-api/src/main/java/org/apache/nifi/components/PropertyDescriptor.java

```diff
@@ -719,6 +720,15 @@ public boolean isSensitive() {
         return sensitive;
     }
 
+    // Pattern to match a parameter reference i.e. "#{anything}"
+    private static final Pattern PARAMETER_REFERENCE = Pattern.compile("^#\\{.*}$");
+
+    public static boolean isSensitiveValueSafeToDisplay(String value) {
+        // If the value is a parameter reference, then it is safe to display the parameter name.
+        // A parameter name is safe to display to users because the sensitive info is stored in the parameter value.
+        return value != null && PARAMETER_REFERENCE.matcher(value).matches();
+    }
```

Comment:
This information should not be part of the `nifi-api` because the persistence strategy is specific to the framework implementation.

Reply:
Or if you prefer to make the change to show ******* instead of Sensitive value set in both the new AND current UIs, then that is fine with me also.

Reply:
I think the new UI should also be showing the same `Sensitive value set` messaging. I don't think moving away from that behavior was intended. I'd need to revisit it to be sure but I would assume we'd want to retain that behavior, when not referencing a Parameter that is.

Reply:
I get your point. Thanks for the clarification that ***** in the dialog is not intended, @mcgilman.